lib/twilio/audiohelper.ts

```diff
@@ -399,6 +410,19 @@ class AudioHelper extends EventEmitter {
     });
   }
 
+  /**
+   * Update AudioHelper options that can be changed by the user
+   * @private
+   */
+  _updateUserOptions(options: AudioHelper.Options): void {
```

Comment:
```suggestion
  private _updateUserOptions(options: AudioHelper.Options): void {
```

Reply:
This is public internal. Meaning, customers cannot use it (not visible in public API), but internal classes can. See device.